README.md

```diff
@@ -10,10 +10,10 @@ The `spv-wallet-web-backend` is an HTTP server that serves as the referential ba
 
 ### Endpoints documentation
 
-For endpoints documentation you can visit swagger which is exposed on port 8080 by default.
+For endpoints documentation you can visit swagger which is exposed on port 5000 by default.
```

Comment:
Do we need to update something in spv-wallet-web-frontend?

Reply:
Yes. Will update too